chat-widget/src/plugins/createChatTranscript.ts

```diff
@@ -80,13 +109,53 @@ class TranscriptHTMLBuilder {
         return htmlData;
     }
 
+    createScriptElement(src: string, integrity: string | undefined = undefined, crossOrigin: string | undefined = undefined, referrerPolicy: string | undefined = undefined) {
+        return `<script src="${src}" ${integrity ? `integrity="${integrity}"`: ""} ${crossOrigin ? `crossorigin="${crossOrigin}"`: ""} ${referrerPolicy ? `referrerpolicy="${referrerPolicy}"`: ""}><\/script>`;
+    }

```

Comment:
I wonder if we should not include the script attribute that is not defined, or we should pick default when undefined

Reply:
Validations are already done in `createExternalScriptElements()` to ensure `src` is defined. Otherwise, the default script is used.